components/Hero/Hero.jsx

```diff
@@ -0,0 +1,27 @@
+import styles from './hero.module.scss'
+import  Wave  from '../Wave/Wave'
+
+const Hero = () => {
+    return (
+      <div className={styles.hero__content_blue}>
```

Comment:
.hero ?

Reply:
?

Reply:
`<div className={styles.hero}>`

Im not sure i fully understand how it works, but I believe this would the correct way of using it?